### PR TITLE
CP-15398: Make texture rebrandable

### PR DESCRIPTION
**Pull request by cheng--zhang**

### CODE REVIEW

**Comment by cheng--zhang** [comment]
Hi @MihaelaStoica 
For: XenAdminTests/SearchTests/ExpectedResults.cs
I just make the test as ignore as you done in indigo because I cannot find out why this problem happens. :-(

I think we can create a CA ticket to track this problem. Do you think so?

File: XenAdmin/Actions/GUIActions/ExternalPluginAction.cs [hunk 1]
```diff
@@ -55,12 +55,12 @@ internal class ExternalPluginAction : AsyncAction
         private const string BlankParamter = "blank";
 
         private static readonly string SnapInTrustedCertXml = 
-            Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), 
-            "Citrix\\XenServerPSSnapIn\\XenServer_Known_Certificates.xml");
+            Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData),
+            Branding.COMPANY_NAME_SHORT + "\\XenServerPSSnapIn\\XenServer_Known_Certificates.xml");
```
Comment (MihaelaStoica) [inline XenAdmin/Actions/GUIActions/ExternalPluginAction.cs]: I think that we can use Path.Combine with 3 parameters here (and on line 63) 
Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), Branding.COMPANY_NAME_SHORT, "XenServerPSSnapIn\\XenServer_Known_Certificates.xml");

File: XenAdmin/Dialogs/OptionsPages/UpdatesOptionsPage.resx [hunk 2]
```diff
@@ -232,7 +232,7 @@
     <value>0</value>
   </data>
   <data name="AllowXenServerUpdatesCheckBox.Text" xml:space="preserve">
-    <value>Check for new versions of Xen&amp;Server</value>
+    <value>Check for &amp;new versions of @BRANDING_SERVER@</value>
```
Comment (MihaelaStoica) [inline XenAdmin/Dialogs/OptionsPages/UpdatesOptionsPage.resx]: This duplicates the hotkey "n" (already used in AllowXenCenterUpdatesCheckBox). I suggest "v" for this one:#
Check for new &amp;versions of @BRANDING_SERVER@

**Comment by MihaelaStoica** [comment]
Looks good. Merging.

**Pull request merged**